### DISCUSSION TITLE
New release 0.18?

### DISCUSSION THREAD

**Discussion by JustAndreww** [zeux/meshoptimizer#402] [General]
Hello!

It's been a while since 0.17, is there any plans to new release? :)

Thank you!

**Comment by zeux** [comment]
Not currently planned, 0.17 was just a couple months ago and the number of changes doesn't quite warrant a new release yet. You can always grab latest binaries from master from artifacts on any of the builds here https://github.com/zeux/meshoptimizer/actions

**Reply by necros2k7** [reply]
Pls tell how to jump exactly to binaries in Actions tab

**Reply by maxxnino** [reply]
In the Artifacts section
https://github.com/zeux/meshoptimizer/actions/runs/1781975705#artifacts
